src/main/java/seedu/address/logic/commands/add/AddCommandAbstract.java

```diff
@@ -0,0 +1,25 @@
+package seedu.address.logic.commands.add;
+
+import seedu.address.logic.commands.Command;
+
+/**
+ * Represents and Add Command for Items.
+ */
+public abstract class AddCommandAbstract extends Command {
+    public static final String COMMAND_WORD = "add";
```

Comment:
might want to maintain consistency with other files where you leave a line at line 9 for this class.